crates/findex/src/ffi/api.rs

```diff
@@ -129,6 +127,37 @@ pub unsafe extern "C" fn h_search(
     )
 }
 
+#[no_mangle]
+#[tracing::instrument(ret)]
+pub unsafe extern "C" fn h_search_with_logs(
+    search_results_ptr: *mut c_char,
+    search_results_len: *mut c_int,
+    master_key_ptr: *const c_char,
+    master_key_len: c_int,
+    label_ptr: *const u8,
+    label_len: c_int,
+    keywords_ptr: *const c_char,
+    entry_table_number: c_uint,
+    progress_callback: ProgressCallback,
+    fetch_entry_callback: FetchEntryTableCallback,
+    fetch_chain_callback: FetchChainTableCallback,
+) -> c_int {
+    log_init("info", 1);
```

Comment:
Don't we want logging by default ? If yes, writing this wrapper can be avoided by adding the log initialization directly in `h_search`.

Reply:
- I didn't want to decrease performances (logs can be intrusive on a huge database)
- for one our client, on iOS, enabling logs was difficult ... so I decided to add explicit functions to do this...

Reply:
Would it be possible to switch at compile time? That way clients would just have to change the release binary for the debug binary in order to activate the logs.

Reply:
Indeed we could use `#[cfg(debug_assertions)]` to init logs and remove all `*_with_logs` functions. However, by mistake `debug` binaries could be pushed in production. What do you think?

Reply:
removing `*_with_logs` functions and build in debug and release modes